collection-scripts/gather_monitoring

```diff
@@ -5,25 +5,88 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-BASE_COLLECTION_PATH="../must-gather"
-MONITORING_PATH="${BASE_COLLECTION_PATH}/monitoring"
+declare -r BASE_COLLECTION_PATH="../must-gather"
+declare -r MONITORING_PATH="${BASE_COLLECTION_PATH}/monitoring"
+declare -r CA_BUNDLE="${MONITORING_PATH}/ca-bundle.crt"
 
-mkdir -p "${MONITORING_PATH}"
+init() {
+  mkdir -p "${MONITORING_PATH}"
 
-MONITORING_ROUTE="$(oc get routes -n openshift-monitoring prometheus-k8s -o jsonpath={.status.ingress[0].host})"
-SA_TOKEN="$(oc sa get-token default)"
+  PROMETHEUS_ROUTE="$(oc get routes \
+    -n openshift-monitoring prometheus-k8s \
+    -o jsonpath='{.status.ingress[0].host}')"
 
-# this is a CA bundle we need to verify the monitoring route, we will write it to disk so we can use it in the flag
-oc -n openshift-config-managed get cm default-ingress-cert -o jsonpath='{.data.ca-bundle\.crt}' > "${MONITORING_PATH}/ca-bundle.crt"
+  ALERT_MANAGER_ROUTE="$(oc get routes \
+    -n openshift-monitoring alertmanager-main \
+    -o jsonpath='{.status.ingress[0].host}')"
 
-# using oc get --raw because we directly control it and have standardized debugging on it
-oc get\
-  --server="https://${MONITORING_ROUTE}"\
-  --token="${SA_TOKEN}" \
-  --certificate-authority="${MONITORING_PATH}/ca-bundle.crt" \
-  --raw=/api/v1/rules?type=alert 2>"${MONITORING_PATH}/alert.stderr" > "${MONITORING_PATH}/alerts.json"
+  SA_TOKEN="$(oc sa get-token default)"
```

Comment:
Is this needed? must-gather is given 'cluster-admin' by overriding the default service account, in the newly created namespace. 

https://github.com/openshift/oc/blob/master/pkg/cli/admin/mustgather/mustgather.go#L564-L585

In short; by getting this SA; and creating [35-40](https://github.com/openshift/must-gather/pull/234/files#diff-a34d5a93515e38dfc675ecd82be2d0c92c9a384475fc1a6b157b311a262daebeR35) we are just complicating the understanding of this script.

Reply:
Makes me wonder why it is part of the current implementation 🤔 .  (@deads2k )
I will dig deeper why this is required.